File: CodeForces/1360a/main.cpp
#include <iostream>

int main()
{

    int n;
    std::cin >> n;
    
    for(int i = 0; i < n; i++)
    {
        int a, b;
        std::cin >> a >> b;
        int c = std::min(std::max(2*a, b), std::max(a, 2*b));
        std::cout << c * c << std::endl;

    }

    return 0;
}

File: CodeForces/1360a/main.py
def inp():
    return(int(input()))
def inlt():
    return(list(map(int,input().split())))


def solve() :
    [a, b] = inlt()
    print(min(max(2 * a, b), max(a, 2 * b)) ** 2)


t = inp()

for _ in range(t) :
    solve()
